tests/examples/python/helm/requirements.txt

```diff
@@ -1,2 +1,2 @@
-pulumi>=0.17.1,<2.0.0
-pulumi_random>=0.5.6,<0.6.0
+pulumi>=1.0.0b3,<2.0.0
+pulumi_random>=1.0.0b1,<2.0.0
```

Comment:
I think this should actually be `pulumi-random` https://pypi.org/project/pulumi-random/1.0.0b1/

Reply:
Doesn't seem like it.  that causes:

```
pulumi:providers:random default_1_0_1  error: no resource plugin 'random-v1.0.1' found in the workspace or on your $PATH, install the plugin using `pulumi plugin install resource random v1.0.1`

[ tes/tests/examples/python/helm ]  +  pulumi:pulumi:Stack helm-p-it-travis-job-helm-1fae8433 create 1 warning

[ tes/tests/examples/python/helm ]     pulumi:providers:random default_1_0_1  1 error
```